### PR TITLE
Add a new application use case

### PR DESCRIPTION
**Pull request by spshin3**
- Propose to add a new application use case for detecting fake video.
- Add a reference for the use case.

***
<a href="https://pr-preview.s3.amazonaws.com/spshin3/webnn/pull/182.html" title="Last updated on Jun 28, 2021, 5:05 AM UTC (e100a27)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/182/75909b7...spshin3:e100a27.html" title="Last updated on Jun 28, 2021, 5:05 AM UTC (e100a27)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -312,6 +312,15 @@ noise suppression using Recurrent Neural Network such as [[RNNoise]] for
...
 audio experiences in video conferences.
 
+### Detecting fake video ### {#usecase-detecting-fake-video}
+
+A user is exposed to realistic fake videos generated by ‘deepfake’ on the web. 
+The fake video can swap the speaker’s face into the president’s face to incite 
+a user politically or to manipulate user’s opinion. The deepfake detection 
+applications such as [[FaceForensics++]] analyze the videos and protect a user against 
+the fake videos or images. When she watches a fake video on the web, the 
+detection application alert her the fraud video in real-time.
```
Comment (anssiko) [inline index.bs:322]:
```suggestion
detection application alerts her of the fraud video in real-time.
```

**Review by anssiko — APPROVED** [review]
Thanks @spshin3! This use case explains how ML can be a solution to an ethically questionable use case ("deepfakes") enabled by the same technology, and as such, sets an aspirational use case target for the group.

This use case also contributes to our work on [documenting ethical issues](https://www.w3.org/2021/04/web-machine-learning-charter.html#ig-other-deliverables), a work item we are planning to start soon in this WG.

I submitted a minor typo fix suggestion, please feel free to incorporate it by committing the suggestion into this branch.

Maybe @wchao1115 can offer a perspective on the feasibility of implementation, see https://arxiv.org/abs/1901.08971

**Review by wchao1115 — APPROVED** [review]

**Review by huningxin — APPROVED** [review]
Thanks @spshin3 ! It looks good with a nit @anssiko already pointed out.

File: index.bs (hunk 1 above)
Comment (spshin3) [reply index.bs:322]: Thanks for review and comments!
And I hope to contribute on ethical and societal issues for WebNN.

**Comment by spshin3** [comment]
Thanks @huningxin ! I have done for the @anssiko 's suggestion.

**Comment by anssiko** [comment]
@wchao1115 @huningxin feel free to merge.

**Pull request merged**